packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -215,13 +215,18 @@ class MediaQueryData {
   ///   property and how it relates to [padding] and [viewInsets].
   final EdgeInsets viewPadding;
 
-  /// The parts of the display that contain system gestures, typically swipes
-  /// from the edge of the screen for navigation.
+  /// The parts of the display where the system may be detecting particular
```

Comment:
Maybe:

The parts of the display where the system may be detecting certain gestures for system actions.

Those gestures are typically drags from the edge of the screen to perform system actions like opening the app drawer or the notification tray. Gestures reserved for system actions may not be forwarded to the app when performed in the area specified by these insets . Therefore, elements placed in these areas may not properly respond to them.

Reply:
I tweaked the example a little (replaced the app drawer with back navigation), but incorporated everything else in your suggestion